### PR TITLE
feat(navigation): iterm support back

### PR DESCRIPTION
**Pull request by japanese-goblinn**
## Description

Bring back support of iTerm but in separate script (promised it in #58)

## Type of change

- [x] Improvement of an existing script

## Screenshot

Screen Shot 2020-11-15 at 01 13 42

### CODE REVIEW

**Comment by PitNikola** [comment]
Hm, I guess it's one of those cases where iTerm is kind of a dependency, but an obvious one which is mentioned in the name (same as Spotify). We should rephrase our guidelines a bit to make it more clear what is considered to be a dependency. Otherwise we would need to handle cases when iTerm is not installed [per our new guidelines](https://github.com/raycast/script-commands/blob/master/CONTRIBUTING.md#scripts-that-require-installation-of-dependencies). Btw, do you know if there is a way to check if an app is installed via Apple Script?

**Comment by thomaspaulmann** [comment]
Yeah, seems fine to not mention `iTerm` as a dependency in the comments of the script command. @japanese-goblinn could you rebase your branch and adjust to the restructuring that has landed on master. Your command should go into `./commands/navigation`. 

It would be cool to use `iTerm`'s app icon. I extracted it from the bundle and added it here. Feel free to add it yourself.
iterm-logo

**Review by thomaspaulmann — CHANGES_REQUESTED** [review]
Please rebase your branch and adopt the latest restructuring. Your command should go into `commands/navigation`. Thanks 🙌

**Comment by japanese-goblinn** [comment]
> Please rebase your branch and adopt the latest restructuring. Your command should go into `commands/navigation`. Thanks 🙌

yep, I'll do it soon

File: commands/navigation/open-finder-from-iterm.applescript [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/osascript
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Open current iTerm directory in Finder
```
Comment (thomaspaulmann) [inline commands/navigation/open-finder-from-iterm.applescript:5]: Let's make it title-cased to align with the other commands.
```suggestion
# @raycast.title Open Current iTerm Directory in Finder
```

File: commands/navigation/open-finder-from-iterm.applescript [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+# @raycast.mode silent
+#
+# Optional parameters:
+# @raycast.icon images/iterm-logo.png
```
Comment (thomaspaulmann) [inline commands/navigation/open-finder-from-iterm.applescript:9]: Thanks for adding the logo. Now it's nicely integrated and feels like an extension of iTerm.

File: commands/navigation/open-finder-from-iterm.applescript [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+#
+# Documentation:
+# @raycast.description Open curren iTerm directory in Finder
```
Comment (thomaspaulmann) [inline commands/navigation/open-finder-from-iterm.applescript:12]: Small typo
```suggestion
# @raycast.description Open the current iTerm directory in Finder
```

File: commands/navigation/open-iterm-from-finder.applescript [hunk 4]
```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/osascript
+
+# Required parameters:
+# @raycast.schemaVersion 1
+# @raycast.title Open current Finder directory in iTerm
```
Comment (thomaspaulmann) [inline commands/navigation/open-iterm-from-finder.applescript:5]: Same as above.
```suggestion
# @raycast.title Open Current Finder Directory in iTerm
```

File: commands/navigation/open-iterm-from-finder.applescript [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+# @raycast.mode silent
+#
+# Optional parameters:
+# @raycast.icon images/iterm-logo.png
+#
+# Documentation:
+# @raycast.description Open curren Finder directory in iTerm
```
Comment (thomaspaulmann) [inline commands/navigation/open-iterm-from-finder.applescript:12]:
```suggestion
# @raycast.description Open the current Finder directory in iTerm
```

**Review by thomaspaulmann — CHANGES_REQUESTED** [review]
Thanks for adding the iTerm icon. Now it feels nicely integrated. Let's make the user-visible strings title-cased to look between the other commands. Otherwise it's good to go.

**Comment by japanese-goblinn** [comment]
@thomaspaulmann done

**Comment by japanese-goblinn** [comment]
@thomaspaulmann really confused who this curren was and how she appeared in script descriptions

**Pull request merged**